Translators/WZDx/main.py

```diff
@@ -21,60 +20,17 @@ def record_active(feature):
     else:
         return False
 
-def update_sat_region_name(request, tim_body):
-    new_tim = copy.deepcopy(tim_body)
-    region_name = new_tim['dataframes'][0]['regions'][0]['name']
-    region_name = region_name.replace(
-        'IDENTIFIER', f"SAT_{request['sdw']['recordId']}")
-    if len(region_name) > 63:
-        region_name = region_name[:60] + '...'
-    new_tim['dataframes'][0]['regions'][0]['name'] = region_name
-    return new_tim
-
-
-def update_rsu_region_name(request, tim_body):
-    new_tim = copy.deepcopy(tim_body)
-    region_name = new_tim['dataframes'][0]['regions'][0]['name']
-    region_name = region_name.replace(
-        'IDENTIFIER', f"RSU_{request['rsus'][0]['rsuTarget']}")
-    if len(region_name) > 63:
-        region_name = region_name[:60] + '...'
-    new_tim['dataframes'][0]['regions'][0]['name'] = region_name
-    return new_tim
-
-
-def translate(wzdx_geojson):
-    tims = []
-    duration = os.getenv("DURATION_TIME", 30)
-    # if no RSUs found, drop that one
-    for feature in wzdx_geojson:
-        tim_body = generate_tim(feature)
-        if tim_body is not None:
-            for msg in tim_body["dataframes"]:
-                # update start date to include milliseconds if missing
-                if msg["startDateTime"][-5] != ".":
-                    msg["startDateTime"] = msg["startDateTime"][:-1] + ".000Z"
-                # set duration time
-                msg["durationTime"] = duration
-
-            sdx_request = get_sdw_request(feature["geometry"])
-            sdx_tim = {
-                "request": sdx_request,
-                "tim": update_sat_region_name(sdx_request, tim_body)
-            }
-            tims.append(sdx_tim)
-
-            rsu_request = get_rsu_request(feature)
-            if rsu_request is not None:
-                rsu_tim = {
-                    "request": rsu_request,
-                    "tim": update_rsu_region_name(rsu_request, tim_body)
-                }
-                tims.append(rsu_tim)
-        else:
-            logging.info(f'Failed to generate TIM for feature: {feature["id"]}')
-    return tims
-
+def delete_tims(feature_list):

```

Comment:
Doesn't look like we use feature, can we leave it out?

Reply:
The feature_list is used when determining which TIMs are still present in the WZDx feed unless I'm misunderstanding this